pkg/kclient/deployments.go

```diff
@@ -32,3 +38,25 @@ func (c *Client) CreateDeployment(deploymentSpec appsv1.DeploymentSpec) (*appsv1
 	}
 	return deploy, nil
 }
+
+// UpdateDeployment updates a deployment based on the given deployment spec
+func (c *Client) UpdateDeployment(name string, deploymentSpec appsv1.DeploymentSpec) (*appsv1.Deployment, error) {
```

Comment:
can `UpdateDeployment()` be like `CreateDeployment()` and just take `deploymentSpec` as an argument? We can get the name from the objectMeta `objectMeta.Name` like in `CreateDeployment()`

Reply:
Hmm, thought I had fixed that. Nice catch! Fixed.